llzk/src/dialect/function/ops.rs

```diff
@@ -106,6 +110,26 @@ pub trait FuncDefOpLike<'c: 'a, 'a>: OperationLike<'c, 'a> {
         unsafe { llzkFuncDefOpGetIsStructConstrain(self.to_raw()) }
     }
 
+    /// If the function name is [`FUNC_NAME_COMPUTE`](llzk_sys::FUNC_NAME_COMPUTE), return the "self"
+    /// value (i.e. the return value) from the function. Otherwise, return Err(ExpectedMthdName).
+    fn self_value_of_compute(&self) -> Result<Value<'c, 'a>, Error> {
+        if self.name_is_compute() {
+            Ok(unsafe { Value::from_raw(llzkFuncDefOpGetSelfValueFromCompute(self.to_raw())) })
+        } else {
+            Err(Error::ExpectedMthdName(&llzk_sys::FUNC_NAME_COMPUTE))
```

Comment:
NitL `ExpectedMethodName` is not that much longer than `ExpectedMthdName`, I'd prefer if we just used that.

Reply:
Since we call these functions everywhere else I'll just change it to that